geth/api/app_state.go

```diff
@@ -15,7 +15,7 @@ func (a AppState) String() string {
 // Specific app states
 // see https://facebook.github.io/react-native/docs/appstate.html
 const (
-	AppStateForeground = AppState("foreground")
+	AppStateActive     = AppState("active")
```

Comment:
In case of `active` vs `inactive` what is `background` then? IMHO the old naming did match better.

Reply:
agree, it is React naming being stupid.
I'll rename the constant back, but keep the "active" value to be in-sync with React Native.